src/Groups/GAPGroups.jl

```diff
@@ -178,41 +302,33 @@ end
 
 Base.:*(x::GAPGroupElem, y::GAPGroupElem) = _prod(x, y)
 
-function ==(x::PermGroup, y::PermGroup)
-   return x.X == y.X && x.deg == y.deg
-end
+==(x::PermGroup, y::PermGroup) = x.X == y.X && x.deg == y.deg
 
-function ==(x::GAPGroup, y::GAPGroup)
-   return x.X == y.X
-end
+==(x::GAPGroup, y::GAPGroup) = x.X == y.X
 
-function ==(x::PermGroupElem, y::PermGroupElem)
-   return x.X == y.X && degree(parent(x))==degree(parent(y))
-end
+==(x::PermGroupElem, y::PermGroupElem) = x.X == y.X && degree(parent(x))==degree(parent(y))
```

Comment:
I realize you are just refactoring the existing code, but while we are at it: perhaps we should first check the degree (as it ought to be fast to get) before comparing the permutations? I.e.,
```suggestion
==(x::PermGroupElem, y::PermGroupElem) = degree(parent(x))==degree(parent(y)) && x.X == y.X
```

Reply:
Yes.